docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3'
+
+services:
+###> doctrine/doctrine-bundle ###
+    database:
+        image: mysql:${MYSQL_VERSION:-8}
+        environment:
+            MYSQL_DATABASE: ${MYSQL_DATABASE:-su_myapp}
+            # You should definitely change the password in production
+            MYSQL_ROOT_PASSWORD: ${MYSQL_ROOT_PASSWORD:-ChangeMe}
+        volumes:
+            - db-data:/var/lib/mysql
+            # You may use a bind-mounted host directory instead, so that it is harder to accidentally remove the volume and lose all your data!
+            # - ./docker/db/data:/var/lib/mysql:rw
+###< doctrine/doctrine-bundle ###
+
+volumes:
+###> doctrine/doctrine-bundle ###
+    db-data:
```

Comment:
I didn't run this. @wachterjohannes could you test if this is all fine?
Config is based postgresql config provided by the recipe: https://github.com/sulu/skeleton/pull/144/files

Reply:
@alexander-schranz looks good had tested this a few days ago with a fresh installation. but would you go with postgre or mysql?

Reply:
I would say our community is more on mysql. /cc @chirimoya